posta/utils.py

```diff
@@ -9,4 +10,4 @@ def imposta_destinatari_e_scrivi_messaggio(request, qs_destinatari=Persona.objec
     qs_destinatari = Persona.objects.filter(pk__in=qs_destinatari.values_list('id', flat=True))
     request.session["messaggio_destinatari"] = qs_destinatari
     request.session["messaggio_destinatari_timestamp"] = datetime.now()
-    return redirect("/posta/scrivi/")
+    return redirect("{}?da_elenco=1".format(reverse('posta-scrivi')))
```

Comment:
Rispettosamente dissento :) Proprio perché non è fattibile / sensato un refactoring unico sto progressivamente portando il codice all'uso di reverse. I vantaggi in termini di robustezza e semplicità sono tali (oltre ad essere il metodo "standard") secondo me da vincere anche sulla consistenza (con questo metodo ho già individuato / corretto alcuni bachetti nel mentre che sistemavo il resto :) )